code/modules/mob/living/simple_animal/abnormality/waw/clown_smiling.dm

```diff
@@ -20,7 +20,7 @@
 	see_in_dark = 10
 	stat_attack = HARD_CRIT
 	speed = 2
-	move_to_delay = 2
+	move_to_delay = 3
```

Comment:
Try 2.3 or 2.5

Reply:
On the map on which i tested it as a clerk, it still seemed a bit too fast; unfarily so. I think this is fine for now, but if player feedback suggests it became too slow I will use your suggestion.